charts/images.yaml

```diff
@@ -22,20 +22,20 @@ images:
   - name: hubble-ui
     sourceRepository: github.com/cilium/hubble-ui
     repository: quay.io/cilium/hubble-ui
-    tag: v0.7.3
+    tag: v0.7.9
   - name: hubble-ui-backend
     sourceRepository: github.com/cilium/hubble-ui-backend
     repository: quay.io/cilium/hubble-ui-backend
-    tag: v0.7.3
+    tag: v0.7.9
   - name: envoy
     sourceRepository: github.com/envoyproxy/envoy
     repository: docker.io/envoyproxy/envoy
-    tag: v1.14.5
+    tag: v1.18.2
```

Comment:
There is v1.18.3, see https://github.com/envoyproxy/envoy/releases/tag/v1.18.3

Reply:
There is but is not used in the latest cilium release v1.10.1